security/cert_manager_operator/cert-manager-authenticate.adoc

```diff
@@ -0,0 +1,33 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cert-manager-authenticate-aws"]
+= Authenticating the {cert-manager-operator}
+include::_attributes/common-attributes.adoc[]
+:context: cert-manager-authenticate
+
+toc::[]
+
+You can authenticate the cert-manager Operator for Red Hat OpenShift on the cluster by configuring the cloud credentials.
+
+// with AWS Security Token Service
+include::modules/cert-manager-configure-cloud-credentials-aws-sts.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_cert-manager-authenticate-gcp"]
+.Additional resources
+* xref:../../installing/installing_aws/installing-aws-customizations.adoc#cco-ccoctl-configuring_installing-aws-customizations[Configuring the Cloud Credential Operator utility]
+
+// with GCP Workload Identity
+include::modules/cert-manager-configure-cloud-credentials-gcp-sts.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+[id="additional-resources_cert-manager-authenticate-gcp-workload-identity"]
+.Additional resources
+* xref:../../installing/installing_gcp/installing-gcp-customizations.adoc#cco-ccoctl-configuring_installing-gcp-customizations[Configuring the Cloud Credential Operator utility]
+
+//  on AWS
+include::modules/cert-manager-configure-cloud-credentials-aws-non-sts.adoc[leveloffset=+1]
+
+// on GCP
+include::modules/cert-manager-configure-cloud-credentials-gcp-non-sts.adoc[leveloffset=+1]
+
```

Comment:
It's good to place the most common way for users in the first position.

Suggested orders:
```
Authenticating on AWS
Authenticating with AWS Security Token Service
Authenticating on GCP
Authenticating with GCP Workload Identity
```
or
```
Authenticating on AWS
Authenticating on GCP
Authenticating with AWS Security Token Service
Authenticating with GCP Workload Identity
```
@swghosh does it make sense?

Reply:
good suggestion.I have opted for the former option, Swarup, let me know otherwise.

Reply:
@lunarwhite I have pished the changes. Can you take another look and approve if looks good?

Reply:
OK with me.
/lgtm